Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1068,9 +1068,21 @@
     <!-- YTXT: Body text for for results next steps  -->
     <string name="submission_test_result_steps_added_body">"Your test has been stored in the Corona-Warn-App."</string>
     <!-- XHED: Page headline for pending test result next steps  -->
-    <string name="submission_test_result_pending_steps_waiting_heading">"Your test result is not available yet."</string>
+    <string name="submission_test_result_pending_steps_waiting_pcr_heading">"Your test result is not available yet."</string>
```

Comment:
Just add the strings here, and do not change the old identifiers. With that you don't have to change other string files.